.github/workflows/buf.yaml

```diff
@@ -42,6 +42,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: bufbuild/buf-setup-action@v1
+        with:
+          github_token: ${{ github.token }}
```

Comment:
Reason?

Reply:
The wf was flaky due to rate-limiting (unauthed requests)